pkg/api/api.go

```diff
@@ -343,8 +343,9 @@ func (hs *HTTPServer) registerRoutes() {
 			alertsRoute.Get("/states-for-dashboard", Wrap(GetAlertStatesForDashboard))
 		})
 
-		apiRoute.Get("/alert-notifications", Wrap(GetAlertNotifications))
-		apiRoute.Get("/alert-notifiers", Wrap(GetAlertNotifiers))
+		apiRoute.Get("/alert-notification-channels", Wrap(GetAlertNotificationChannels))
+		apiRoute.Get("/alert-notifications", reqEditorRole, Wrap(GetAlertNotifications))
+		apiRoute.Get("/alert-notifiers", reqEditorRole, Wrap(GetAlertNotifiers))
```

Comment:
Noticed this endpoint is only used when editing alerting notifcation channels so required at least editor for that one. Can probable move down the GET /alert-notifications to the /alert-notifications group.